instrumentation/github.com/bradfitz/gomemcache/memcache/otelmemcache/config.go

```diff
@@ -18,25 +18,11 @@ import (
 	oteltrace "go.opentelemetry.io/otel/trace"
 )
 
-type config struct {
-	serviceName    string
-	tracerProvider oteltrace.TracerProvider
-}
-
 // Option is used to configure the client.
-type Option func(*config)
+type Option oteltrace.TracerProvider
```

Comment:
We should probably stick with a generic Option type like we had before.  Even though we only have a single option now, we may want to add others in the future.

Reply:
Yes, now I see this could potentially be a breaking change, good point :+1:. I'll keep the type as it was.